pkgs/development/libraries/libfaketime/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./no-date-in-gzip-man-page.patch
+    ./nix-store-date.patch
```

Comment:
Or use `substituteAll` function here to replace the placeholder directly in the patch.